format/src/main/scala/org/scalafmt/internal/AssociatedTrivias.scala

```diff
@@ -0,0 +1,99 @@
+package org.scalafmt.internal
+
+import org.scalafmt.internal.TokensOps._
+
+import scala.meta._
+import scala.meta.Token
+import scala.meta.Token.Comment
+import scala.meta.contrib._
+
+import org.scalameta.logger
+
+final case class AssociatedTrivias(
+    leadings: Map[Token, Tokens],
+    trailings: Map[Token, Tokens]
+) {
+  private def pretty(token: Token): String = {
+    if (token.is[Token.BOF]) {
+      "BOF"
```

Comment:
could be `␂`, but it does not show on my terminal with the font I have installed.